functions/source/rest_client.py

```diff
@@ -275,6 +280,63 @@ def create_workspaces(account_id, workspace_name, deployment_name, aws_region, c
         print('workspace FAILED about to raise an exception')
         raise Exception(workspace_status_message)
 
+
+    if workspace_status == 'RUNNING' and hipaa_parm == 'No':
+        # api-endpoint
+        URL = "https://"+deployment_name+".cloud.databricks.com/api/2.0/policies/clusters/create"
+
+        # Json data
+        DATA = {
+            "name": "Default Cluster Policy",
+            "definition": "{\"spark_version\":{\"type\":\"fixed\",\"value\":\"next-major-version-scala2.12\",\"hidden\":true}}"
+        }
+
+        print(DATA)
+        response_policy = post_request(URL, DATA, encodedbase64, user_agent, version)
+        print(response_policy)
+        # parse the policy_id element from the response
+        policy_id = response_policy['policy_id']
+
+        response.update(response_policy)
+        print(response)
+
+        # DEFAULT CLUSTER
```

Comment:
Let's add a comment that this cluster being created is starting in a terminating state